google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkServiceUserSettings.java

```diff
@@ -91,6 +96,22 @@ void setEnableAutomaticUpdates(boolean enableAutomaticUpdates) {
         DEFAULT_MANAGED_SDK_AUTOMATIC_UPDATES /* need to specify default to avoid removal */);
   }
 
+  long getLastAutomaticUpdateTimestamp() {
```

Comment:
would it make sense to return an Optional boxed Long here instead to indicate that there is no previously stored last update?

Reply:
I agree, good use case for Optional, let's do that